testing/cage/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Maintainer: Will Sinatra <wpsinatra@gmail.com>
+# Contributor: Will Sinatra <wpsinatra@gmail.com>
+pkgname=cage
+pkgver=0.1.1
+pkgrel=0
+pkgdesc="A Wayland Kiosk"
+url="https://github.com/Hjdskes/cage"
+arch="x86_64"
```

Comment:
Is it really x86_64 only ?

Reply:
Right you are, looks like it builds on all arches.